src/txkube/_compat.py

```diff
@@ -17,3 +17,13 @@ def dumps_bytes(obj):
     if isinstance(b, unicode):
         b = b.encode("ascii")
     return b
+
+
+
+def ensure_unicode(s, encoding="ascii", errors="strict"):
+    """
+    Ensure that ``s`` is ``unicode``.  Decode if it is bytes.
```

Comment:
What if it's `None`?  Any other non-bytes type?

Reply:
I based this function on the `ensure_str` function used in the `syx` library at lyft:
  https://github.com/lyft/syx/blob/master/syx/__init__.py#L11

So `None` or non-bytes types will just be returned.

Reply:
Just because Lyft does it doesn't make it a good idea.  Can you explain why this behavior is acceptable?